lib/ImageProcessor.js

```diff
@@ -58,25 +59,27 @@ class ImageProcessor {
         if ( config.exists("backup") ) {
             const backup = config.get("backup");
             backup.acl = backup.acl || acl;
+            backup.cacheControl = (backup.cacheControl !== undefined) ? backup.cacheControl : cacheControl;
             backup.bucket = backup.bucket || bucket;
 
             promise = promise
               .then(() => this.execBackupImage(backup, imageData))
-              .then((image) => this.fileSystem.putObject(image))
+              .then((image) => this.fileSystem.putObject(image, backup))
```

Comment:
`backup` will have much information. I suggest `{cacheControl: backup.cacheControl}` is enough to pass and makes clearly.

Reply:
I thought this could be an opening for other options `putObject` can look at. Maybe put it under `"metadata": {...}`? (as well as the `acl` which is pretty much the same - it's currently put in `ImageData` even though it doesn't really make sense there? not sure)

Reply:
I agree. `metadata` seems good idea to set S3 attributes.